packages/contracts-bedrock/test/invariants/interop/PROPERTIES.md

```diff
@@ -0,0 +1,46 @@
+| 13  | SupERC20      | The `SuperchainTokenBridge#relayERC20()` function MUST only process messages originating from the `L2toL2CrossDomainMessenger`.                                                                                          | [ ]    |
+| 14  | SupERC20      | `ETHLiquidity#mint()` MUST never be callable such that balance would decrease below `0`.                                                                                                                                 | [ ]    |
+| 15  | SupERC20      | `ETHLiquidity#burn()` MUST never be callable such that balance would increase beyond `type(uint256).max`.                                                                                                                | [ ]    |
+| 16  | SharedLockbox | The total withdrawable ETH amount present on all the dependency set chains MUST NEVER be more than the amount held by the `SharedLockbox` of the cluster.                                                                | [ ]    |
+| 17  | SharedLockbox | The `OptimismPortal` MUST lock the ETH amount on the `SharedLockbox` when on a deposit transaction with value greater than zero, without holding any ETH balance from the depositing users.                              | [ ]    |
+| 18  | SharedLockbox | The `OptimismPortal` MUST unlock the ETH amount being withdrawn from the `SharedLockbox` if it is greater than zero.                                                                                                     | [ ]    |
+| 19  | SharedLockbox | The `SharedLockbox` MUST NOT trigger a new deposit transaction when unlocking ETH from the `OptimismPortal`.                                                                                                             | [ ]    |
+| 20  | SharedLockbox | Once `SuperchainConfig#addChain` is called, a message to each chain on the dependency set MUST be sent to include the input chain, as well as a message to the input chain for each current chain on the dependency set. | [ ]    |
+| 21  | SharedLockbox | The chain MUST be added to the dependency set on L2 as long as the config deposit transaction and the caller are valid.                                                                                                  | [ ]    |
+| 22  | SharedLockbox | Valid messages MUST revert if the message comes from a chain that is not in the destination chain's dependency set.                                                                                                      | [ ]    |
+| 23  | SharedLockbox | `SuperchainConfig#addChain` MUST revert if the new chain dependency set size is greater than zero.                                                                                                                       | [ ]    |
```

Comment:
For now we won't have the possibility to check a specific dependency set from a chain